### ISSUE TITLE
如何查看eval loss

### ISSUE THREAD

**Issue by microchila** [modelscope/ms-swift#6917]
nproc_per_node=4
nohup bash -c "
NPROC_PER_NODE=$nproc_per_node \
CUDA_VISIBLE_DEVICES=0,1,2,3 \
swift sft \
    --model /home/dugis/shihaofeng/llm/Qwen3-8B \
    --train_type lora \
    --dataset '/home/dugis/shihaofeng/llm/ali_address_res.jsonl' \
    --load_from_cache_file true \
    --torch_dtype float16 \
    --num_train_epochs 1 \
    --per_device_train_batch_size 1 \
    --per_device_eval_batch_size 1 \
    --learning_rate 1e-4 \
    --gradient_accumulation_steps $(expr 16 / $nproc_per_node) \
    --lora_rank 8 \
    --lora_alpha 32 \
    --target_modules all-linear \
    --eval_steps 100 \
    --save_steps 100 \
    --save_total_limit 20 \
    --logging_steps 5 \
    --max_length 2048 \
    --output_dir output \
    --loss_scale ignore_empty_think \
    --warmup_ratio 0.05 \
    --deepspeed zero2 \
    --report_to swanlab \
    --swanlab_project swift-robot \
    --swanlab_token 'egCV96nZ8JNC0ui9mLbBi'
    --dataloader_num_workers 4 "  > train.log 2>&1 &

**Comment by Jintao-Huang** [comment]
--split_dataset_ratio or --val_dataset
